### PR TITLE
Component/search bar

### PR DESCRIPTION
**Pull request by hebernard**

### CODE REVIEW

File: catalog/search-bar/index.html [hunk 1]
```diff
@@ -18,25 +18,37 @@
...
   <body>
+  <div class="wrapper">
     <pearson-search-bar placeholder="Search here" startdisabled>
       <ul slot="search-options">
         <li id="optionOne">first option</li>
         <li id="optionTwo">second option</li>
         <li id="optionThree">third option</li>
       </ul>
-      <pearson-dropdown buttonText="Please Select" type="icon" slot="filter">
+      <search-bar-filter buttonText="Please Select" type="icon" slot="filter">
```
Comment (davodey) [inline catalog/search-bar/index.html]: @hebernard  I dont think we need the buttonText or type attribute.  They type attribute adds a class of 'icon-btn-i8' to the button in the dropdown menu.  lets hardcode that in.  Let me know if you have questions.

File: catalog/search-bar/webpack.config.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
   mode: 'production',
-  entry: ['./js/dist/pearson-search-bar.js', './js/dist/pearson-dropdown.js'],
+  entry: ['./js/dist/pearson-search-bar.js', './js/dist/search-bar-filter.js'],
```
Comment (davodey) [inline catalog/search-bar/webpack.config.js]: @hebernard I know you worked hard at the webpack solution, but Im pretty sure this is duplicating something we already have.  Does this concat the filter and the search component js?

File: catalog/search-bar/index.html (hunk 1 above)
Comment (hebernard) [reply catalog/search-bar/index.html]: @davodey makes sense, will fix

File: catalog/search-bar/webpack.config.js (hunk 2 above)
Comment (hebernard) [reply catalog/search-bar/webpack.config.js]: @davodey Yes. We should take a look at your original next time we meet. I could not get it to work for me and so I just did this as a work around.

Comment (davodey) [reply catalog/search-bar/webpack.config.js]: @hebernard sounds good!

**Pull request closed without merging**